### PR TITLE
Added frenchify, copied nearly exactly from the php frenchify

### PR DESCRIPTION
**Pull request by TheScriptPony**
frenchify is a lot of lines, hence all the changes. Also modified bify a little bit, so that's the removed stuff

### CODE REVIEW

**Comment by TheScriptPony** [comment]
Wait do I need to update the info.json to include the guy who made the php?

**Comment by ShroomAgent27** [comment]
Crediting the original author would be appreciated, yes. Personally, I'd suggest editing the info.json to include the author's name, and include a comment in the copied code that source where it came from.

In addition, make sure that you have permission to copy code. We're [GPL3](https://github.com/brandons209/Red-bot-Cogs/blob/v3/LICENSE), so make sure the place you are copying from allows us to use it. You can ask if you can't figure it out, just drop a link to where you got it from.

File: memeify/memeify.py [hunk 1]
```diff
@@ -56,11 +69,216 @@ def __bify(self, bify_str, cmd) -> str:
...
+        else:
+            await ctx.send(self.__french_pre_f(*msg_c))
+
+    def __french_pre_f(self, french, cmd):
+        emoji_list = []
+        if cmd:
+            french_cmd_fix = french.split(" ")
+            french_str = " ".join(french_cmd_fix[1:])
+
+        emoji_match = re.finditer(custom_emoji, french_str)
```
Comment (brandons209) [inline memeify/memeify.py:99]: Big error. `french_str` can be referenced before assignment if cmd is not none. You need an else for this statement to sent french_str (or move this under the if statement.

Comment (brandons209) [reply memeify/memeify.py:99]: Happens when you run frenchify with no text.

**Review by brandons209 — CHANGES_REQUESTED** [review]

**Review by brandons209 — APPROVED** [review]
Alright, looks good!

**Pull request merged**